test/Datadog.Trace.Tests/RuntimeMetrics/RuntimeMetricsWriterTests.cs

```diff
@@ -113,7 +114,9 @@ public void CleanupResources()
                 // ignored
             }
 
-            Assert.True(writer.ExceptionCounts.IsEmpty);
+            writer.ExceptionCounts.TryGetValue(nameof(CustomException1), out var count);
```

Comment:
@kevingosse Sorry for being late to the party but you could define a new Exception type that's only used in this test case, instead of using `CustomException1` which is used in this test case and the `ShouldCaptureFirstChanceExceptions` test case